ethereum/migrations/prod/12_pyth_set_fee_1_wei.js

```diff
@@ -0,0 +1,21 @@
+const dotenv = require("dotenv")
+dotenv.config({ path: "../../.env" });
+
+if (process.env.CLUSTER !== undefined) {
+  dotenv.config({ path: `../../.env.cluster.${process.env.CLUSTER}`});
+}
+
+const setFeeVaa = process.env.MIGRATION_12_SET_FEE_VAA;
```

Comment:
is this going to go into the dotenv file and be stored permanently? (I think that it should be)

Reply:
Yes it's gonna be there forever. In general I like keeping all the changes to contract.